packages/api/blocks-api/src/blocks/createRichTextBlock.ts

```diff
@@ -70,12 +71,10 @@ export interface RichTextBlockInputInterface<LinkBlockInput extends BlockInputIn
 export function createRichTextBlock<LinkBlock extends Block>({
     link: LinkBlock,
     indexSearchText = true,
+    migrateOptions = { migrations: [], version: 0 },
 }: CreateRichTextBlockOptions): Block<RichTextBlockDataInterface, RichTextBlockInputInterface<ExtractBlockInput<LinkBlock>>> {
     const BLOCK_NAME = "RichText";
-    const MIGRATE: MigrateOptions = {
-        migrations: [],
-        version: 0,
-    }; // Placeholder for future migrations
```

Comment:
@nsams we have this placeholder in some of our factories, but not in all. What do you think about it? One case where this might come in handy is when we move away from storing Draft.js state. We could then have a migration in the library and migrate all our applications automatically. When we allow custom migrations here, we eliminate this option...

Reply:
Generally, library blocks also need to have the possiblity for migrations, they need to get merged somehow. Don't we do that for any block so far? And do we have not a single migration in library so far?

Reply:
> Don't we do that for any block so far? And do we have not a single migration in library so far?

Nope (as far as I know)

We once discussed changing the migrations form a version to a timestamp approach, but we've never done that.

Reply:
ah, yes, that would be the solution for correctly merging.

Reply:
@nsams I've created a task for this. We definitely need this once we switch RTE, so it will be done sooner than later 😁